### DISCUSSION TITLE
View timeline in frames or seconds?

### DISCUSSION THREAD

**Discussion by Vcclark** [keiwando/toonsquid#270] [Q&A]
Is It possible to change the default workflow once a project has been started? In other words, can the timeline view be changed from frames to seconds and back again on an existing project ?

**Comment by DWrightAnimation** [comment]
As far as I know, if you go into the settings, you can change the number of frames a single drawing creates from 120 to 1, and probably back again. Also, the timeline is mainly zoomed out, to see the frames, just zoom in, to see seconds again, just zoom out.
